_docs/_latest/use/explain.md

```diff
@@ -0,0 +1,274 @@
+```
+ 
+With resource expansion enabled, the example above will return `organizations/1234567890` and all folders, projects, VMs, and other resources in it.
+
+To constrain the result to a certain type of permission, use the following command:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_member user/<USER_NAME> iam.roles.get
+```
+
+The above query will list all resources that can be accessed by the member with a binding of `iam.roles.get`.
+
+#### List resource permissions
+
+The following command lists all members that can access a resource by a direct binding:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME>
+```
+
+Similar to the `access_by_member` command, group expansion is not performed by default. For example, `group/bar` has `roles/owner` on the `organization/1234567890` and no other policies. Without group expansion, the following example will only return `group/bar`:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource organizations/1234567890
+```
+
+To enable group expansion pass in the `--expand_group` argument. Forseti will perform group expansion and list all members that can access the resource, even if the access is granted because a member is in a group.
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME> --expand_group true 
+```
+
+With group expansion enabled, the example above will list `group/bar` and `user/foo` if `user/foo` is a member in `group/bar`.
+
+To constrain the result to a certain type of permission, pass in the permission:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME> <PERMISSION> 
+```
+
+#### Access member or resource by permission
+
+You can specify a permission, such as `p`, and list `<members, resource, role>` pairs that have a relation.
+
+For instance a that role contains permission p and member has permission p on resource.
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_authz --permission iam.serviceAccounts.get
+...
+Member member1
+Member member2
+Resource resource1
+Role roles/owner
+...
+```
+
+These results mean that resource `resource1` has a policy to grant `roles/owner` to `member1` and `member2`
+
+With `access_by_authz` you can also specify role instead of permission 
+
+```
+# With G Suite Group expansion
+$ forseti-client-XXXX-vm> forseti explainer access_by_authz --expand_group
+
+# With resource expansion.
+$ forseti-client-XXXX-vm> forseti explainer access_by_authz --expand_resource
+```
+
+#### View permission source
+Understand why a member has a permission to a resource
+
+```
+$ forseti-client-XXXX-vm> forseti explainer why_granted <MEMBER_NAME> <RESOURCE_NAME> --permission <PERMISSION_NAME>
+```
+
+Example values for `<MEMBER_NAME>`, `user/user1@gmail.com`, `serviceAccount/service1@domain.com`
+
+#### Understand why a member doesn’t have a permission on a resource
```

Comment:
It seems like this is less about understanding why they don't have permission, and instead showing the bindings they need to add to grant the permission?

Reply:
Updated.